packages/manager/apps/web/client/app/hosting/multisite/git-association/hosting-multisite-git-association.html

```diff
@@ -0,0 +1,136 @@
+<oui-message data-type="error" data-ng-if="$ctrl.errorMessage">
+    <small data-ng-bind-html="$ctrl.errorMessage"></small>
+</oui-message>
+
+<h1 data-translate="hosting_multisite_git_association_title"></h1>
+<p data-translate="hosting_multisite_git_association_configure_description"></p>
+
+<form
+    class="form-inline"
+    name="$ctrl.gitAssociationForm"
+    data-ng-submit="$ctrl.generateLink()"
+>
+    <h3 data-translate="hosting_multisite_git_association_ssh_key"></h3>
+    <p
+        data-translate="hosting_multisite_git_association_ssh_key_description"
+    ></p>
+    <div class="row">
+        <oui-field
+            class="col-md-4"
+            data-label="{{ ::'hosting_multisite_git_association_ssh_key' | translate }}"
+        >
+            <oui-clipboard
+                data-ng-if="$ctrl.sshKey"
+                data-model="$ctrl.sshKey"
+            ></oui-clipboard>
+        </oui-field>
+    </div>
+    <h3 data-translate="hosting_multisite_git_association_configure"></h3>
+
+    <oui-message type="info">
+        <p
+            class="font-weight-bold"
+            data-translate="hosting_multisite_git_association_configure_info_title"
+        ></p>
+        <ul>
+            <li
+                data-translate="hosting_multisite_git_association_configure_info_description"
+            ></li>
+            <li
+                data-translate="hosting_multisite_git_association_configure_info_https"
+            ></li>
+            <li
+                data-translate="hosting_multisite_git_association_configure_info_ssh"
+            ></li>
+        </ul>
+        <span
+            class="bold"
+            data-translate="hosting_multisite_git_association_configure_info_empty_repo"
+        ></span>
+        <span
+            data-translate="hosting_multisite_git_association_configure_info_not_empty_repo"
+        ></span>
+        <br />
+        <br />
+        <p
+            class="font-weight-bold"
+            data-translate="hosting_multisite_git_association_configure_info_add_in_paremeters"
+        ></p>
+    </oui-message>
+
+    <div class="row">
+        <oui-field
+            class="col-md-4"
+            data-label="{{ ::'hosting_multisite_git_association_repository' | translate }}"
+        >
+            <div class="d-flex oui-input-group oui-input-group_inline">
+                <input
+                    type="text"
+                    id="repository_url"
+                    name="repository_url"
+                    class="oui-input mr-2"
+                    data-ng-pattern="/^[https:\/\/|git:\/\/].*/"
```

Comment:
```suggestion
                    data-ng-pattern="$ctrl.MY_SUPER_REGEX"
```
You can move this in constant file.
In addition, i think we can put ```<input type="url">```